release-notes.md

```diff
@@ -4,6 +4,14 @@
 
 The following updates were made in this release:
 
+**Android Identity 1.1.1**
+
+- Identity extension now ignores any custom identifiers with null or empty ids when calling `syncIdentifier` or `syncIdentifiers` APIs as the Visitor ID Service does not support these identifiers. 
+- Identity extension now ignores the `syncIdentifiers` APIs call with an empty Map. 
```

Comment:
"API", not "APIs", or reword the sentence like I did in the previous comment.

Reply:
Please rewrod so that it reads as follows:

"The `syncIdentifiers` API call is ignored when there is an empty Map."